File: C# .NET Core/CarComponents/Engine.cs
namespace CarCatalog.CarComponents
{
    public class Engine
    {
        public readonly Cylinders Size;
        public readonly int HorsePower;
        public readonly int CityMPG;
        public readonly int HighwayMPG;
    
        public Engine(Cylinders cylinders, int horsePower, int cityMPG, int hwyMPG)
        {
            Size = cylinders;
            HorsePower = horsePower;
            CityMPG = cityMPG;
            HighwayMPG = hwyMPG;
        }

        public enum Cylinders
        {
            I4 = 4,
            V6 = 6,
            V8 = 8
        }
    }
}

File: Python/vehicle_components/Engine.py
class Engine(object):
    """description of class"""
    def __init__(self, model_number, cylinders, horse_power, city_mpg, hwy_mpg):
        self.model_number = model_number
        self.cylinders = cylinders
        self.horse_power = horse_power
        self.city_mpg = city_mpg
        self.hwy_mpg = hwy_mpg
        

    def get_details(self):
        print("Vehicle Engine Details")
        print("Model No.: {}".format(self.model_number))
        print("Cylinders: {}".format(self.cylinders))
        print("Horsepower: {}".format(self.horse_power))
        print("City MPG: {}".format(self.city_mpg))
        print("Highway MPG: {}".format(self.hwy_mpg))
